package.json

```diff
@@ -61,10 +62,10 @@
     "gulp-load-plugins": "^1.2.4",
     "gulp-util": "^3.0.7",
     "gzip-js": "~0.3.2",
-    "jest": "^15.1.1",
-    "jest-config": "^15.1.1",
-    "jest-jasmine2": "^15.1.1",
-    "jest-runtime": "^15.1.1",
+    "jest-cli": "17.0.2",
```

Comment:
Sorry, can you change it back to `jest`? It's being used here: https://github.com/facebook/react/blob/64cba04bf976f2f11b428bb9ec1108cd2da70e53/scripts/fiber/record-tests#L11